sense/downstream_tasks/gesture_recognition/__init__.py

```diff
@@ -32,3 +32,33 @@
 }
 
 INT2LAB = {value: key for key, value in LAB2INT.items()}
+
+LAB_THRESHOLDS = {
+    "\"Drinking\" gesture": .2,
+    "\"Sleeping\" gesture": .7,
+    "Calling someone closer": .5,
+    "Covering ears": .5,
+    "Covering eyes": .5,
+    "Dabbing": .5,
+    "Facepalming": .5,
+    "Nodding": .5,
+    "Pointing left": .3,
+    "Pointing right": .3,
+    "Pointing to the camera": .5,
+    "Putting finger to mouth": .5,
+    "Rolling hand": .5,
+    "Scratching": .5,
+    "Shaking head": .5,
+    "Showing the middle finger": .3,
+    "Swiping down": .4,
+    "Swiping down (with two hands)": .5,
+    "Swiping left": .5,
+    "Swiping right": .5,
+    "Swiping up": .5,
```

Comment:
Swiping down often triggered immediately after swiping up, because I just lowered my hand again

Reply:
Increase `Swiping down` up to `.7`? I've always had this issue. Is this more of a model not learning from the full action issue or?

Reply:
I think a higher threshold might help. Give it a try

Reply:
I tried up to .8, which is too high for a single `Swipe down`. Currently at .65 it is still not perfect because the issue you described is still triggered very often :confused:

Reply:
Ok, then that's just how it is